examples/Extensions/AddSitelinks.php

```diff
@@ -0,0 +1,339 @@
+                ),
+                self::createAdScheduleInfo(
+                    DayOfWeek::WEDNESDAY,
+                    18,
+                    MinuteOfHour::ZERO,
+                    21,
+                    MinuteOfHour::ZERO
+                ),
+                self::createAdScheduleInfo(
+                    DayOfWeek::THURSDAY,
+                    18,
+                    MinuteOfHour::ZERO,
+                    21,
+                    MinuteOfHour::ZERO
+                ),
+                self::createAdScheduleInfo(
+                    DayOfWeek::FRIDAY,
+                    18,
+                    MinuteOfHour::ZERO,
+                    21,
+                    MinuteOfHour::ZERO
+                )
+            ]
+        ]);
+
+        // Issues a mutate request to add the extension feed items.
+        $extensionFeedItemServiceClient = $googleAdsClient->getExtensionFeedItemServiceClient();
+        $response = $extensionFeedItemServiceClient->mutateExtensionFeedItems(
+            $customerId,
+            [
+                new ExtensionFeedItemOperation(['create' => $extensionFeedItem1]),
+                new ExtensionFeedItemOperation(['create' => $extensionFeedItem2]),
+                new ExtensionFeedItemOperation(['create' => $extensionFeedItem3]),
+                new ExtensionFeedItemOperation(['create' => $extensionFeedItem4])
+            ]
+        );
+
+        // Prints some information about the created extension feed items.
+        printf(
+            "Created %d extension feed items with the following resource names:%s",
+            $response->getResults()->count(),
+            PHP_EOL
+        );
+        $createdExtensionFeedItemsResourceNames = [];
+        foreach ($response->getResults() as $addedExtensionFeedItem) {
+            /** @var ExtensionFeedItem $addedExtensionFeedItem */
+            print "{$addedExtensionFeedItem->getResourceName()}" . PHP_EOL;
+            $createdExtensionFeedItemsResourceNames[] = $addedExtensionFeedItem->getResourceName();
+        }
+        return $createdExtensionFeedItemsResourceNames;
+    }
+
+    /**
+     * Creates a new site link feed item with the specified attributes.
+     *
+     * @param $sitelinkText string the text of the sitelink feed item
+     * @param $sitelinkUrl string the URL of the sitelink feed item
+     * @return SitelinkFeedItem the created site link feed item
+     */
+    private static function createSitelinkFeedItem(string $sitelinkText, string $sitelinkUrl)
+    {
+        return new SitelinkFeedItem([
+            'link_text' => new StringValue(['value' => $sitelinkText]),
+            'final_urls' => [new StringValue(['value' => $sitelinkUrl])],
+        ]);
+    }
+
+    /**
+     * Creates a new AdScheduleInfo with the specified attributes.
+     *
+     * @param int $day the enum value of day of the week of the AdScheduleInfo
+     * @param int $startHour the starting hour of the AdScheduleInfo
```

Comment:
Is there any enum we could use like for the minute instead?

Reply:
Nope. I guess it's too much to create an enum of 24 hours.
That for minute actually has four values for four quarters.